test/javascript/components/ResourceSection.test.js

```diff
@@ -0,0 +1,51 @@
+import { parseHTML } from '../test_helpers';
+import { mount,
+         createLocalVue } from '@vue/test-utils';
```

Comment:
Does this need to be run to install it? `yarn add --dev @vue/test-utils`

Not sure because I ran yarn install and then I ran that later and it `saved 1 new dependency`
I was just looking at the `test-utils` ReadMe